src/garage/_dtypes.py

```diff
@@ -935,6 +940,16 @@ def to_time_step_list(self):
             })
         return samples
 
+    def get_terminals(self):
```

Comment:
I don't think you need `get_` here, it's perfectly clean and nicer looking to just have this method named `.terminals`. It could also reasonably be a property, but I don't have a strong opinion about that.